lost/src/main/java/com/mapzen/android/lost/internal/MockEngine.java

```diff
@@ -131,6 +132,11 @@ private Location nodeToLocation(NodeList nodeList, NodeList speedList, int i) {
             location.setTime(System.currentTimeMillis());
             location.setSpeed(Float.parseFloat(speedList.item(i).getFirstChild().getNodeValue()));
 
+            if (previous != null) {
+                location.setBearing(previous.bearingTo(location));
```

Comment:
If there is no `previous` value the bearing is not set and `Location#hasBearing()` will return `false`.